src/types/monaco.d.ts

```diff
@@ -0,0 +1,9 @@
+import { IRange } from 'monaco-editor'
+
+declare module 'monaco-editor' {
+    namespace editor {
+        interface IStandaloneCodeEditor {
+            setHiddenAreas(ranges: IRange[]): void
+        }
+    }
+}
```

Comment:
Is this supposed to be temporary ? Is there an issue filed on monaco repo ? If yes maybe add a comment

Reply:
Nop, it's not temporary, the method was make public by our custom build of monaco

=> https://github.com/CodinGame/monaco-editor-wrapper/pull/57/commits/e02f1a59b79fcb417bcc3b01dd7bb2d094b674f4